server/routes/userRoute.js

```diff
@@ -99,7 +99,9 @@ router.put("/:id/picUpload", authorize, async (req, res, next) => {
   const image_upload = upload.single('image');
```

Comment:
What's the API request that you are making to test this? it should have a file in 'image'